src/component/Calculator.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
```

Comment:
- [ ] Since it is a requirement for this project and best practice, that is each component should perform a specific task, thus making your code reusable, please create a `Calculator.js` component separate file and import it into the `App.js` component. See screenshot for more clarifications. 

![Calculator](https://user-images.githubusercontent.com/62849872/153070277-f5a9559e-158c-48c5-8922-c4e056f82407.png)